pages/common/doctl-auth.md

```diff
@@ -0,0 +1,24 @@
+# doctl auth
+
+> Authenticate doctl with one or more API tokens.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/auth/>.
+
+- Show available commands:
+
+`doctl auth --help`
+
+- Open a prompt to enter an API token and label its context:
+
+`doctl auth init --context {{token_label}}`
+
+- List authentication contexts (API tokens):
+
+`doctl auth list`
+
+- Switch contexts (API tokens):
+
+`doctl auth switch --context {{token_label}}`
+
+- Remove a context (API token):
```

Comment:
Not sure I understand the `(API token)` in brackets here. Can we reword this to improve clarity? Same goes with the example above this one.

Reply:
That's a good point. In Digital Ocean, when you generate an API token, enter it into doctl, and give it a label, it creates a context.

When switching contexts you are switching between stored API tokens. Does the following read, better, or is there a better way to word this?
`- Remove a context (a locally stored API token):`

Reply:
```suggestion
- Remove a stored authentication context (API token):
```
Does this illustrate what I wrote above?

Reply:
Ah, I see! Thanks for clarifying. Yeah, @258204 - that looks good!